packages/arb-token-bridge-ui/synpress.config.ts

```diff
@@ -48,6 +50,28 @@ export default defineConfig({
       ) =>
         TestWETH9__factory.connect(tokenAddress, testWallet.connect(provider))
 
+      // Deploy test ERC-20 token
+      const erc20Contract = new TestERC20__factory().connect(
+        wallet.connect(ethProvider)
+      )
+      const l1TestErc20Token = await erc20Contract.deploy()
+      await l1TestErc20Token.deployed()
+
+      const l2Network = await getL2Network(wallet.connect(arbProvider))
```

Comment:
Currently this will throw an error because our local network doesn't exist in the L2 network list in the sdk.

Waiting for an input from the team how to best handle it.